src/Sylius/Bundle/CoreBundle/CommandHandler/VariantsUpdateHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\CommandHandler;
+
+use Sylius\Bundle\CoreBundle\Applicator\CatalogPromotionApplicatorInterface;
+use Sylius\Bundle\PromotionBundle\Provider\EligibleCatalogPromotionsProviderInterface;
+use Sylius\Component\Core\Repository\ProductVariantRepositoryInterface;
+use Sylius\Component\Product\Command\UpdateBatchedVariants;
+use Webmozart\Assert\Assert;
+
+final class VariantsUpdateHandler
```

Comment:
Change the name of this handler to be the same as name of the command with `Handler` suffix

Reply:
Or change the name of the command - what I would say is a better thing to do

Reply:
👍🏻 for changing the name of the command, but probably the name of the handler should be also changed to follow the pattern `CommandName` + `Handler`